{"uuid":"21b166bd-767a-40d9-bc68-92d207d2a2db","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["a4c32593-dcca-4635-a110-f9d1a5b94df4"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681591057573,"stop":1681591057573},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681591058362,"stop":1681591058362},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"aa8549f2-9a1b-41f8-9c3b-21ab54ed58d5-attachment","type":"image/png"},{"name":"Myscreenshot","source":"065485ce-9bd0-49ab-a1d7-71b576c79368-attachment"}],"parameters":[],"start":1681591068487,"stop":1681591068718},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681591068718,"stop":1681591068718},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681591068719,"stop":1681591068719}],"start":1681591057350,"stop":1681591068719}